apps/site/assets/ts/stop/__tests__/components/StationInformationTest.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import StationInformation from "../../components/StationInformation";
+import { Stop } from "../../../__v3api";
+
+const stationStop = { "station?": true } as Stop;
+const busStop = { "station?": false } as Stop;
+
+describe("StationInformation", () => {
+  it("should have headings", () => {
+    render(<StationInformation stop={stationStop} />);
+    expect(
+      screen.queryByRole("heading", { name: "Station Information" })
+    ).toBeInTheDocument();
+    expect(
+      screen.queryByRole("heading", { name: "Bringing Your Car or Bike" })
+    ).toBeInTheDocument();
+    expect(
+      screen.queryByRole("heading", { name: "Getting Around the Station" })
+    ).toBeInTheDocument();
+    expect(
+      screen.queryByRole("heading", { name: "Purchasing Fares" })
+    ).toBeInTheDocument();
+  });
+
+  it("hides unneeded headings for bus stops", () => {
+    render(<StationInformation stop={busStop} />);
+    expect(
+      screen.queryByRole("heading", { name: "Stop Information" })
+    ).toBeInTheDocument();
+    expect(
+      screen.queryByRole("heading", { name: "Bringing Your Car or Bike" })
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole("heading", { name: "Getting Around the Station" })
+    ).toBeInTheDocument();
```

Comment:
Shouldn't this not be in the document. It's not on the bus stop page when viewing so curious that this is passing.

Reply:
It should be on the bus stop page. It's the heading for the Accessibility amenity card (which isn't hidden on bus stops).

Reply:
On the figma mocks Accessibility and Fares are both under Station Information without any subheaders. Could check with design if that's the desired display